CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ Changelog
 .. note:: This version is not yet released and is under active development.
 
 * Added support for Python 3.6.
+* General performance increase when encoding/decoding ASN.1 structures.
```

Comment:
Oh. One more thing. The README should indicate that there was a change in dependency.